packages/ember-data/lib/system/model/states.js

```diff
@@ -343,7 +345,7 @@ var DirtyState = {
 
     rolledBack: function(record) {
       get(record, 'errors').clear();
-      record.triggerLater('ready');
+      record.transitionTo('loaded.saved');
```

Comment:
@turboMaCk not needed, I have already a fix available, just haven't made the PR yet.